### DISCUSSION TITLE
Contribute to wiki

### DISCUSSION THREAD

**Discussion by Quantum-Codes** [CubeyTheCube/scratchclient#22] [Q&A]
How do i contribute? Change setting to allow everyone to make?

**Comment by CubeyTheCube** [comment]
just edit the `docs` folder and PR

**Reply by Quantum-Codes** [reply]
> just edit the `docs` folder and PR

The docs folder just has 2 lines...
I want to update for example... the messages page

Forking doesn't fork the wiki
